sound/soc/intel/boards/sof_sdw.c

```diff
@@ -198,6 +198,7 @@ static const struct dmi_system_id sof_sdw_quirk_table[] = {
 		},
 		.driver_data = (void *)(SOF_RT711_JD_SRC_JD1 |
 					SOF_SDW_TGL_HDMI |
+					SOF_RT715_DAI_ID_FIX |
 					SOF_SDW_PCH_DMIC),
```

Comment:
Is it correct that we have both `SOF_RT715_DAI_ID_FIX` and `SOF_SDW_PCH_DMIC`?

Reply:
@bardliao SOF_RT715_DAI_ID_FIX is only used in rt715(-sdca). 
1. If the platform is using PCH DMIC without RT715, the SOF_SDW_PCH_DMIC works and SOF_RT715_DAI_ID_FIX won't take any effects.
2. If the platform is using  RT715 without PCH DMIC, Pierre's recent patch will ignore SOF_SDW_PCH_DMIC. So it is still OK.
3. If the platform is using RT715 + DMIC. The hardware design conflicts with our PCM assignment. We should not use such design.
4. If the platform has neither RT715 nor SOF_SDW_PCH_DMIC, SOF_RT715_DAI_ID_FIX still be OK as it won't take effects. SOF_SDW_PCH_DMIC may have some issues. But as the DMI is ADL RVP, this situation won't happen. ADL RVP will always have either PCH_DMIC or RT715.

Reply:
@libinyang Thanks for the explanation. I missed Pierre's patch. I have no concern now.